### ISSUE TITLE
Why not merge in the 3dickulus version?

### ISSUE THREAD

**Issue by micahscopes** [Syntopia/Fragmentarium#26]
3dickulus' fork is really wonderful and has proven to be more stable for me.  Is there any particular reason why you haven't merged their changes into the main stream?

**Comment by Atavic** [comment]
There are two different devs involved: 3Dickulus and Benesi.
http://www.digilanti.org/fragmentarium/

**Comment by Syntopia** [comment]
Yes - the other forks are the way to go for updated versions - this branch has been unmaintained for some time, and I do no plan to bring it up to date. Some time ago I added a link to the 3dickulus fork on the main page, in order to encourage people to try out one of the other branches.

**Comment by Atavic** [comment]
:+1: to all the devs for the efforts.
